package.json

```diff
@@ -48,31 +46,20 @@
         "lint:packagejson": "pretty-quick --pattern package.json --pattern \"packages/*/package.json\" --pattern \"projects/*/package.json\" --pattern \"tools/*/package.json\" --pattern \"react/*/package.json\"",
         "lint:ts": "pretty-quick --pattern \"packages/**/*.ts\" && eslint -f pretty \"packages/**/*.ts\" && pretty-quick --pattern \"tools/**/*.ts\" && eslint -f pretty \"tools/**/*.ts\"",
         "lint:versions": "node ./scripts/lint-versions.js",
-        "new-package": "cd projects/templates && plop",
```

Comment:
You can see this updated below, we don't have to move into the directory to run the command since we can leverage `yarn workspaces`